src/groupeddataframe/grouping.jl

```diff
@@ -901,29 +940,55 @@ colwise(fns::Union{AbstractVector, Tuple}, d::AbstractDataFrame) = [f(d[i]) for
 colwise(f, gd::GroupedDataFrame) = [colwise(f, g) for g in gd]
 
 """
-    by(d::AbstractDataFrame, keys, cols => f...; sort::Bool = false)
-    by(d::AbstractDataFrame, keys; (colname = cols => f)..., sort::Bool = false)
+    by(d::AbstractDataFrame, keys, cols => f; sort::Bool = false)
```

Comment:
The signature is supposed to present supported methods, not to give particular examples (which are given at the end). Please use the same presentation strategy as the one I implemented for `combine` above.

Reply:
Added.